src/libraries/System.Security.Cryptography.Csp/src/System/Security/Cryptography/CapiHelper.Windows.cs

```diff
@@ -1283,7 +1283,7 @@ public static bool VerifySign(SafeProvHandle hProv, SafeKeyHandle hKey, int calg
                 }
 
                 // Get the key contents
-                byte[] rgbKey = null;
+                byte[] rgbKey = null!;
```

Comment:
Why is this not nullable?

Reply:
it will always set with non null value within the method below `UnloadKey(hProv, hKey, ref rgbKey, ref cbKey);`, and further usages of `rgbKey` is not expecting null. But it still need to initialized before used in `UnloadKey(...)` method

Reply:
Doesn't adding `NotNull` to the ref parameter address this issue?

Reply:
Yes, forgot about that, thanks!